### DISCUSSION TITLE
Bubble.List显示错位？求助。

### DISCUSSION THREAD

**Discussion by mashupeng1** [ant-design/x#337] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/b1eeb6c4-a092-42b8-a0fb-d27c641af741)
![image](https://github.com/user-attachments/assets/cb717a31-6ffb-4645-a2ed-4926a3555ef0)

    const roles: GetProp<typeof Bubble.List, 'roles'> = {
        ai: {
            placement: 'start',
            typing: {step: 5, interval: 20},
            styles: {
                content: {
                    borderRadius: 16
                }
            },
            avatar: {icon: <UserOutlined/>, style: {background: '#D5D5D5'}},
            footer: <Space>
                <Button color="default" variant="text" size="small" icon={<SyncOutlined/>}/>
                <Button color="default" variant="text" size="small" icon={<CopyOutlined/>}/>
            </Space>
        },
        local: {
            header: "username",
            placement: 'end',
            variant: 'shadow',
            avatar: {icon: <UserOutlined/>, style: {background: '#87D068'}}
        }
    };
    
    local就是右手边的placement: 'end'，加上header，就是显示用户名，这里用假的字符串代替。加上header或者footer，就会显示错位。

**Comment by YumoImer** [comment, marked as answer]
https://github.com/ant-design/x/pull/314

已解决，1.0.3 发布
